src/components/wallet/methods/trustAsset.ts

```diff
@@ -18,24 +18,14 @@ export default async function trustAsset(
   issuer?: string,
   pincode_stretched?: Uint8Array
 ) {
+  this.loading = { ...this.loading, trust: true }
+  this.error = null
+  console.log(this.loading)
+  const finish = () => (this.loading = { ...this.loading, trust: false })
   try {
-    let instructions
-
-    this.logger.instruction(
-      'We need to add a trustline to the asset to ensure the deposit will be expected'
-    )
-
-    if (asset && issuer) {
-      instructions = [asset, issuer]
-      this.logger.instruction(
-        'There is already a trustline on this account, no need to recreate it'
-      )
-    } else {
-      this.logger.instruction(
-        'There isn’t currently a trustline on this account so we need to add one'
-      )
-      instructions = await this.setPrompt({ message: '{Asset} {Issuer}' })
-      instructions = instructions.split(' ')
+    if (!asset || !issuer) {
+      let instructions = await this.setPrompt({ message: '{Asset} {Issuer}' })
+      ;[asset, issuer] = instructions.split(' ')
```

Comment:
wrong line for the semicolon

Reply:
for some reason the auto-formatter really wants it there... we don't usually use semicolons but in this case, where we do a spread and the line starts with an open brackets, its needed. I don't know why the formatter keeps moving it down there